### PR TITLE
Tests jasmine

### PR DESCRIPTION
**Pull request by jffaust**
New pull request for the jasmine tests.
The branch `tests-cucumber` will be closed now.

### CODE REVIEW

File: dist/marionette.gridview.js [hunk 1]
```diff
@@ -77,15 +82,15 @@
...
-      this.autoSave = options.autoSave;
   
-      if (!_.isUndefined(options.autoPos)) {
+      if (_.isUndefined(options.autoPos)) {
         options.autoPos = true;
       }
       if (!options.collection) {
         throw new Error('Missing collection inside initialization options');
       }
       this.options = options;
```
Comment (Nelrohd) [inline dist/marionette.gridview.js]: this is misleading, options was really part of Backbone.View before. Please use something else as `settings`

File: dist/marionette.gridview.js [hunk 2]
```diff
@@ -20,8 +25,8 @@
       name:     DEFAULT_WIDGET_NAME,
       x:        0,
       y:        0,
-      width:    0,
-      height:   0,
+      width:    1,
```
Comment (Nelrohd) [inline dist/marionette.gridview.js]: What is 1? which unit?

File: dist/marionette.gridview.js [hunk 3]
```diff
@@ -51,7 +56,7 @@
   });
   
   GridView.WidgetView = Marionette.ItemView.extend({
-    template: _.template('<div class="some-widget"><p>default view</p></div>'),
+    template: _.template('<div class="default-widget"><p>default view</p></div>'),
```
Comment (Nelrohd) [inline dist/marionette.gridview.js]: Constants for template

File: dist/marionette.gridview.js [hunk 4]
```diff
@@ -65,7 +70,7 @@
   });
   
   GridView.WidgetGridView = Marionette.LayoutView.extend({
-    template: '#gridview-template',
+    template: _.template('<div id="main-gridstack" class="grid-stack">  </div>'),
```
Comment (Nelrohd) [inline dist/marionette.gridview.js]: Same as above

File: dist/marionette.gridview.js (hunk 2 above)
Comment (jffaust) [inline dist/marionette.gridview.js]: Gridstack units, if you init a gridstack with a size of 5x5, you could place 5 widgets with widths of 1

**Comment by jffaust** [comment]
Is that good enough to receive a +1 @Nelrohd  :) ??

**Comment by jffaust** [comment]
@Nelrohd !

**Comment by Nelrohd** [comment]
Tests ran? +1

**Comment by jffaust** [comment]
Now they pass

**Pull request merged**